packages/fxa-content-server/app/scripts/templates/connect_another_device.mustache

```diff
@@ -42,15 +42,17 @@
       {{#isFirefoxDesktop}}
         <p id="install-mobile-firefox-desktop">
           {{#isSignedIn}}
+            <div id="pair-value-prop" class="flex items-center justify-center mt-4 mb-5">
+              <p class="font-normal text-sm w-[261px]">{{#t}}Want to get your tabs, bookmarks, and passwords on another device?{{/t}}</p>
```

Comment:
What's this look like without `[w-261px]`? Generally I've pushed back against explicit widths like this since we don't know how it'll look across locales when it's translated and the copy could change in the future.

Reply:
> What's this look like without `[w-261px]`?

It uses 100% of the width.

> Generally I've pushed back against explicit widths like this since we don't know how it'll look across locales when it's translated and the copy could change in the future.

Doesn't feel like a great process for a design/jira to make it into an active sprint only to have the engineer push back.

As for the copy changing again... yeah I literally worked on this page not long ago.  (You helped with that green check!  😄)

Reply:
> Doesn't feel like a great process for a design/jira to make it into an active sprint only to have the engineer push back.

Yeah, that's true. I've definitely conveyed to Design that we would not be doing this, [this is a long comment](https://mozilla-hub.atlassian.net/browse/FXA-5454?focusedCommentId=564513) but I've got a note there about "Some of the new copy in the designs break mid-sentence...."; in the original "redesign" they had a lot of these and we'd all talked about it and agreed not to set explicit widths, I'd asked if they'd like us to set left/right padding to get it kind of there or not and they've told me they'd prefer to just not have a width or extra padding set based on copy. I'll re-emphasize to them for the future because it's confusing otherwise 🙂

Reply:
Yeah, I used `px-16` in my most recent PR.  And I'm going to use that here as well.